### PR TITLE
fix a few errors in build instructions

### PR DESCRIPTION
**Pull request by oliverkurth**
When trying to build tdnf I found a few minor issues in the README.

### CODE REVIEW

**Comment by keerthanakalyan** [comment]
@oliverkurth : Could you please rebase this PR to the latest commit https://github.com/vmware/tdnf/commit/a9750b233abebe320e3e041dc991c7dde25fefd1, to address the CI/CD failures.

File: README.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ cachedir=/var/cache/tdnf
 Now configure repo files under `/etc/yum.repos.d` or your repodir following
 `.repo` format of dnf/yum.
 
-You should now have a client executable named tdnf under `tools/cli`. To test
+You should now have a client executable named tdnf under `bin/cli`. To test
```
Comment (sidcha) [inline README.md]: The path must just be `bin/` (and not `bin/cli`)

File: README.md [hunk 2]
```diff
@@ -5,18 +5,18 @@
 In order to compile, from the checkout directory, run the following
 
 ```sh
-cmake
+cmake .
```
Comment (sidcha) [inline README.md]: Also can you change this to the cmake convention-conformant, out-of-source builds? such as:

```bash
mkdir build && cd build
cmake ..
make
```

**Comment by oliverkurth** [comment]
Thanks @cbsiddharth , I applied the fixes. A rebase wasn't needed, looks like the tests merge the changes into the `dev` branch.

**Comment by sidcha** [comment]
Initially this got me confused; how did the build succeed without my changed (that fixed it). If the branch is being built, a rebased of this branch is necessary to get things rolling again. But appears travis builds the would-be merge HEAD instead of branch's HEAD.

From travis doc:

> Rather than build the commits that have been pushed to the branch the pull request is from, we build the merge between the source branch and the upstream branch

That explains why the rebase was not needed and its important to keep this in mind.

**Review by sidcha — APPROVED** [review]
LGTM. Thanks!

**Review by keerthanakalyan — APPROVED** [review]

**Pull request merged**